test/test_dpres_plevel.py

```diff
@@ -36,7 +36,8 @@
 pressure_surface_2d_msg = pressure_surface_2d_nan.copy()
 pressure_surface_2d_msg[np.isnan(pressure_surface_2d_msg)] = -99.0
 
-pressure_surface_2d_msg_asfloat32 = pressure_surface_2d_msg.astype(np.float32)
+pressure_surface_2d_msg_asfloat32 = pressure_surface_2d_msg.ast
+ype(np.float32)
```

Comment:
Looks like a typo added a newline in `astype`

Reply:
Oh, turns out this typo was made by my elbow after running all the tests and seeing everything passing, probably due to reaching to my drink just before pushing the changes :) Sorry for the inconvenience, fixed.